presto-main/src/main/java/com/facebook/presto/sql/analyzer/AggregationAnalyzer.java

```diff
@@ -167,9 +173,31 @@ protected Boolean visitAtTimeZone(AtTimeZone node, Void context)
         @Override
         protected Boolean visitSubqueryExpression(SubqueryExpression node, Void context)
         {
+            AstUtils.preOrder(node)
+                    .filter(isInstanceOfAny(DereferenceExpression.class, Identifier.class))
+                    .map(Expression.class::cast)
+                    .forEach(this::analyzeSubqueryReference);
+
             return true;
         }
 
+        private void analyzeSubqueryReference(Expression referenceExpression)
+        {
+            ResolvedField resolvedField = analysis.getScope(referenceExpression)
```

Comment:
The case that might not work:
`SELECT (SELECT a AS a) FROM (VALUES (1,2)) t(a,b) GROUP BY b`